### DISCUSSION TITLE
Support Dock Event hook on Macos or Windows？

### DISCUSSION THREAD

**Discussion by richerfu** [tauri-apps/tauri#5493] [Q&A] [answered]
i set `prevent_exit` when close event trigger.But i can't reopen main window when i click dock icon.
So is there support `dock event` hooks ?

<img width="706" alt="图片" src="https://user-images.githubusercontent.com/32590310/198287792-c9e76166-bea5-4edd-a1fe-9e0a319650c6.png">

**Comment by FabianLars** [comment, marked as answer]
Not yet: https://github.com/tauri-apps/tao/pull/517

**Reply by richerfu** [reply]
Any planning to accpet this PR?

**Reply by FabianLars** [reply]
yes, but no ETA
